README.md

```diff
@@ -34,7 +34,7 @@ You can [download signed release tarballs of AppStream on freedesktop.org](https
 ## Information For Users
 
 ### For Upstream Application Authors
-- **[Quickstart Guides](https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html)**:
+- **[MetaInfo Quickstart Guides](https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html)**:
```

Comment:
The guides have nothing to do with just MetaInfo data - one is explicitly about how to deploy AppStream data if you are a distributor, the other is about translations, and only two are about MetaInfo files. So I think the previous sentence in that context is good enough.

Reply:
But the heading in the page says `Metadata Quickstart`. Should that be updated too ?

![appstream](https://github.com/user-attachments/assets/156600f8-6bbc-4efe-9dd9-b11a15f5ceac)

Reply:
I don't think so... All of this is concerning metadata...